Idref.js

```diff
@@ -2,62 +2,85 @@
 	"translatorID": "271ee1a5-da86-465b-b3a5-eafe7bd3c156",
 	"label": "Idref",
 	"creator": "Sylvain Machefert",
-	"target": "^https?://www\\.idref\\.fr/.*",
+	"target": "^https?://www\\.idref\\.fr/",
 	"minVersion": "3.0",
 	"maxVersion": "",
 	"priority": 100,
 	"inRepository": true,
 	"translatorType": 4,
-	"browserSupport": "g",
-	"lastUpdated": "2018-02-03 09:39:41"
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-02-12 13:03:06"
 }
 
+/*
+   IdRef Translator
+   Copyright (C) 2018 Sylvain Machefert - web@geobib.fr
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
```

Comment:
All Zotero translators should be licensed under AGPL (Affero), see https://github.com/zuphilip/translators/wiki/Common-code-blocks-for-translators for the relevant block.

Reply:
now fixed. I had copied another translator from the repo without having in mind this information regarding the license to be used. AGPL is also fine for me, thanks for the pointer.